packages/core/src/query/flux.ts

```diff
@@ -201,6 +201,8 @@ export function toFluxValue(value: any): string {
     return value.toISOString()
   } else if (value instanceof RegExp) {
     return sanitizeRegExp(value)
+  } else if (Array.isArray(value)) {
+    return `[${value.map(toFluxValue).join(',')}]`
```

Comment:
One thing to be careful about is that an array's type is `object`, and we already have a `typeof value === 'object'` check further up which would be truthy. We don't enter that condition as there is another `&& typeof value[FLUX_VALUE] === 'function'`check there.

It might be slightly more future-proof change the order so we have the `Array.isArray` check first.

However, the tests will probably catch that scenario anyways.

Reply:
Sure, this library completely relies upon tests ... so the existing code works as expected. Yes, there are already 5 recognized object instances, and there could be fewer comparisons at runtime if these are nested under `if object`.  I changed it that way.